dmg/ArangoDB3e-CLI.app/Contents/Resources/arango-dfdb

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
I would vote for DRY here if possible. Pls rather have the file just once, and install it to the respective targets with a loop.